djweb/dj_comp_hist/migrations/0001_initial.py

```diff
@@ -1,4 +1,4 @@
-# Generated by Django 2.1 on 2018-12-07 21:36
+# Generated by Django 2.1.5 on 2019-03-18 18:14
```

Comment:
We're still deciding on what to do with the migrations (i.e. whether to include these). We'll report back in 1-2 hours.